src/server/scripts/Kalimdor/TempleOfAhnQiraj/boss_cthun.cpp

```diff
@@ -581,10 +604,31 @@ struct boss_cthun : public BossAI
 
     private:
         //Out of combat whisper timer
-        uint32 WisperTimer;
+        uint32 _whisperTimer;
 
         //Body Phase
         uint8 _fleshTentaclesKilled;
+
+        //If no-one is outside
+        bool _stomachDeathScheduled;
+
+        //Map variable
+        Map* _map;
```

Comment:
Why?

Reply:
I felt obliged to comment every variable there 😄

Reply:
But why don't you just use me->GetMap(). Defining a map variable here is not needed

Reply:
@kjack9 advised to make a private variable, it looks like an optimisation on matchsticks, but why not?